### ISSUE TITLE
Training and Test Curves on CIFAR-100 (VGG19 and ResNet50)

### ISSUE THREAD

**Issue by rsundar96** [rsundar96/curriculum-learning-acceleration#4]
Following are the training curves obtained when running the Hacochen Curriculum Learning code on CIFAR-100. Following hyper parameters were used to obtain the best possible result that I could obtain.

#### Details on code execution (VGG19):
- Learning Rate: Starts at 0.1 and gradually decreases till 0.001 over equal intervals
- Batch Size: 100 (*default*)
- Epochs: 300
- Optimiser: SGD (*default*)
- Loss Function: Categorical Cross Entropy (*default*)

<img width="724" alt="Screenshot 2020-08-04 at 10 15 12" src="https://user-images.githubusercontent.com/19652780/89276305-68546f80-d63b-11ea-9fdb-b80a8067a453.png">

<img width="724" alt="Screenshot 2020-08-04 at 10 16 21" src="https://user-images.githubusercontent.com/19652780/89276611-c5e8bc00-d63b-11ea-9d71-e0c1babb4491.png">

<img width="724" alt="Screenshot 2020-08-04 at 10 17 46" src="https://user-images.githubusercontent.com/19652780/89276625-ca14d980-d63b-11ea-87d4-fc9d6bdfbe1f.png">

<img width="724" alt="Screenshot 2020-08-04 at 10 18 46" src="https://user-images.githubusercontent.com/19652780/89276727-e87ad500-d63b-11ea-9450-9d762873e860.png">

<img width="724" alt="Screenshot 2020-08-04 at 10 19 58" src="https://user-images.githubusercontent.com/19652780/89276904-16601980-d63c-11ea-804d-43984a54bc78.png">

<img width="724" alt="Screenshot 2020-08-04 at 10 20 51" src="https://user-images.githubusercontent.com/19652780/89277102-4d362f80-d63c-11ea-9001-10287f4de244.png">

**Final Training Accuracy** (Curriculum Learning): 65.0%
**Final Test Accuracy** (Curriculum Learning): 54.97%

**Final Training Accuracy** (Random Initialisation): 55.0%
**Final Test Accuracy** (Random Initialisation): 53.7%

#### Details on code execution (ResNet50):
- Learning Rate: Starts at 0.1 and gradually decreases till 0.001 over equal intervals
- Batch Size: 100 (*default*)
- Epochs: 300
- Optimiser: SGD (*default*)
- Loss Function: Categorical Cross Entropy (*default*)

<img width="724" alt="Screenshot 2020-08-04 at 10 31 43" src="https://user-images.githubusercontent.com/19652780/89278202-b5394580-d63d-11ea-9b9d-e0ca1633b7e8.png">

<img width="724" alt="Screenshot 2020-08-04 at 10 32 27" src="https://user-images.githubusercontent.com/19652780/89278289-d00bba00-d63d-11ea-9198-188bc6c802a7.png">

<img width="728" alt="Screenshot 2020-08-04 at 10 33 26" src="https://user-images.githubusercontent.com/19652780/89278404-f3cf0000-d63d-11ea-8664-ca722bca4136.png">

<img width="728" alt="Screenshot 2020-08-04 at 10 34 13" src="https://user-images.githubusercontent.com/19652780/89278470-0fd2a180-d63e-11ea-98ca-61465ac35af8.png">

<img width="728" alt="Screenshot 2020-08-04 at 10 35 08" src="https://user-images.githubusercontent.com/19652780/89278636-4ad4d500-d63e-11ea-8df8-17d87bed3e82.png">

<img width="728" alt="Screenshot 2020-08-04 at 10 35 54" src="https://user-images.githubusercontent.com/19652780/89278651-4d372f00-d63e-11ea-98fc-261bc81d8942.png">

**Final Training Accuracy** (Curriculum Learning): 61.0%
**Final Test Accuracy** (Curriculum Learning): 55.07%

**Final Training Accuracy** (Random Initialisation): 55.21%
**Final Test Accuracy** (Random Initialisation): 54.7%
